virtcontainers/qemu_arm64.go

```diff
@@ -45,6 +48,65 @@ var supportedQemuMachines = []govmmQemu.Machine{
 	},
 }
 
+// Logger returns a logrus logger appropriate for logging qemu-aarch64 messages
+func qemuArmLogger() *logrus.Entry {
+	return virtLog.WithField("subsystem", "qemu-aarch64")
+}
+
+// On ARM platform, we have different GIC interrupt controllers. Different
+// GIC supports different QEMU parameters for virtual GIC and max VCPUs
+var hostGICVersion = getHostGICVersion()
```

Comment:
I think this should be a local variable in `getGuestGICVersion`

Reply:
@devimc Emm, yoiu are right. I had thought that @Pennyzct would use this variable to adjust the maxvcpu number for QEMU. It seems we are in the same package. I will fix it.

Reply:
@devimc Sorry, I had got a mis-understanding about your comment just now, hostGICVersion would be used in @Pennyzct 's PR https://github.com/kata-containers/runtime/pull/585 later to detect maxvcpu. So I would prefer to keep it as var.